modules/ROOT/pages/overview-of-bonita-bpm-in-a-cluster.adoc

```diff
@@ -83,7 +83,7 @@ Even if your process users do not use Bonita Portal, it is still present in the
 
 The nodes must all be in the same datacenter, to keep latency of updates to shared information to an acceptable level.
 
-All nodes must run the same software configuration (same versions on Bonita and application server).
+All nodes must run the same exact version of the Bonita Platform. e.g. a `7.11.2` will not be able to connect to a cluster of Bonita Platform in `7.11.1`. 
```

Comment:
ℹ️ `Bonita Platform` refers here to the historical name, not the concept described in the 'What is Bonita' page. According to this page, we should use `Bonita Runtime`.
I guess if we want to avoid a naming clash, using `Bonita` should be enough.